docs/chapter2-payload-image-format.rst

```diff
@@ -0,0 +1,670 @@
+                                              Type matches the machine word size. This
+                                              must be provided for the 'firmware' image in
+                                              a configuration, but is optional for others.
+                                              Images without a load address can be loaded
+                                              to any suitable location. They can also be
+                                              left where they are in the FIT, i.e. not
+                                              loaded at all.
+   project           R           string       Type of the image, indicating which project
+                                              produced it \ :sup:`3`:
+
+                                              * "tianocore" - UEFI binary
+                                              * "u-boot" - U-Boot
+                                              * "op-tee" - Open Trusted Execution
+                                                Environment
+                                              * "opensbi" - RISC-V OpenSBI
+                                              * "arm-trusted-firmware" - ARM Trusted
+                                                Firmware
+                                              * “linuxboot” - Linuxboot
+   capabilities      O           string list  List of capabilities that the Payload has:
+                                              (not defined) \ :sup:`4`
+   producer          O           string       Indicates the build system and version
+                                              which produced the FIT
+   uncomp-size       R / O       u32          Size of the uncompressed data in bytes. If
+                                              the data is not compressed, this can be
+                                              omitted.
+   entry-start       O           u32 / u64    If required, this is the offset of the image
+                                              entry point from the load address of the
+                                              image. For example, a value of 0x10 means
+                                              that the image entry point is 16 bytes after
+                                              the start of the image. If omitted, a value
+                                              of 0 is assumed.
+   reloc-start       O           u32 / u64    If the image supports relocation, this is
+                                              the offset of the start of the relocation
+                                              data within the image.
+
+                                              Relocation is described below, in the section
+                                              called"Relocation". This offset can be loaded
+                                              to any word-aligned address. This 'image'
```

Comment:
I think we should remove "This offset.....address.", we do not need to load "reloc table" separately to somewhere word-aligned address.

Reply:
Done! Could you check?